webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -800,9 +800,19 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
 
   validateGradeInput(e) {
 
+    var i18nDecimalPattern = "";
+    /*global portal*/
+    if (this.getDecimalSeparator(portal.locale) === ".") {
+      i18nDecimalPattern = "[\d.]";
+    }
+    else {
+      i18nDecimalPattern = "[\d,]";
+    }
+    var rgxp = new RegExp(i18nDecimalPattern);
+
     if (e.key === "Backspace" || e.key === "ArrowLeft" || e.key === "ArrowRight") {
       return true;
-    } else if (!e.key.match(/[\d.,]/)) {
```

Comment:
I know I'm a little late to this, but don't we just need a + after the closing ]?
/[\d.,]+/

All your change seems to be doing is making the decimal character more specific, based on the portal locale, whereas we should just be able to accept all of the decimal chars.